### DISCUSSION TITLE
StrUtil 增加高级replace

### DISCUSSION THREAD

**Discussion by KING754** [looly/hutool#2623] [Ideas]
增加占位符替换。map,list,obj替换这种。
这样才一些小的场景。有不想用的模板的时候，就非常有用了。

**Comment by looly** [comment]
见StrUtil.format

**Comment by KING754** [comment]
是不是，只有map的？
看看是否可以加list,arr,obj.field

**Reply by looly** [reply]
都有。

**Reply by KING754** [reply]
抱歉，没有仔细看。找到了在：cn.hutool.core.text.StrFormatter
